config/webpack.common.js

```diff
@@ -211,15 +211,6 @@ const config = [
 			poll: 1000,
 		},
 	},
-	{
-		entry: {
-			'eventespresso-core-css-default': pathToEDTRv1 + 'components/ui/styles/themes/default/index.js',
-		},
-		module: moduleConfigWithJsAndCssRules,
-		watchOptions: {
-			poll: 1000,
-		},
-	},
```

Comment:
attempt #2 for:

![image](https://user-images.githubusercontent.com/1477453/79743026-84dacf00-830c-11ea-97c9-f59d0f4f9ba9.png)